subprojects/testing-base/src/test/groovy/org/gradle/api/internal/tasks/testing/logging/GroovyStackTraceSpecTest.groovy

```diff
@@ -74,4 +74,12 @@ class GroovyStackTraceSpecTest extends Specification {
             throw new Exception("boom")
         }
     }
+
+    private static void stackTraceMatches(StackTraceElement element, String className, String methodName, String fileName, int lineNumber) {
```

Comment:
Because `StackTraceElement#equals` doesn't work anymore in JDK 9 : it also compares the class loaders, for example.